challengegroover/routes.py

```diff
@@ -24,10 +32,36 @@ def get_user():
 def callback():
     code = request.values["code"]
     tokens = SpotifyAuth().getUserToken(code)
+    """"get data of new releases and store them"""
+    token = tokens.get('access_token')
+    headers = {'content-type': 'application/json',
+               'Accept': 'application/json',
+               'Authorization': 'Bearer ' + token
```

Comment:
you can have a header builder function as helper to prepare the authorization and renew it if necessary